src/qml/AttachmentsDelegate.qml

```diff
@@ -112,13 +112,35 @@ Column {
         }
     }
 
+    function getMinHeight(attachmentType) {
+        var minHeight = units.gu(5)
+        switch (attachmentType) {
+        case 'audio':
+            minHeight = units.gu(5)
+            break
+        case 'image':
+        case 'video':
+            minHeight = units.gu(14)
+            break
+        case 'vcard':
+            minHeight = units.gu(9.5)
+            break
+        default:
+            minHeight = units.gu(15)
+        }
+
+        return minHeight
+    }
+
     spacing: units.gu(1)
     Repeater {
         id: attachmentsRepeater
 
         Loader {
             id: attachmentLoader
             property bool loaded: status === Loader.Ready
+            height: loaded ? item.height : getMinHeight(modelData.type)
+            width: units.gu(27)
```

Comment:
Where does this width value come from?

Reply:
All delegates have a min wifth set to 27 units. e.g: https://github.com/ubports/messaging-app/blob/xenial/src/qml/AttachmentDelegates/ImageDelegate.qml#L46